packages/common/container-definitions/src/audience.ts

```diff
@@ -5,8 +5,8 @@
 
 // False positive: this is an import from the `events` package, not from Node.
 // eslint-disable-next-line unicorn/prefer-node-protocol
-import { EventEmitter } from "events";
-import { IClient } from "@fluidframework/protocol-definitions";
+import { type EventEmitter } from "events";
+import { type IClient } from "@fluidframework/protocol-definitions";
 
```

Comment:
Is there a tool that is choosing to resolve these with per type `type` imports versus `type` import the whole block (`import type {`)?
I noticed that index.ts imports are using whole block which is the most important IMO. But I like whole block as default so I can see that there really isn't a runtime dep.
If there is a tool with rationale, then I do prefer automation. :)

Reply:
I used the eslint rule's auto-fixer, but I also did some manual tweaks. These are the relevant config settings we currently have:

```json
        "@typescript-eslint/consistent-type-exports": [
            "error"
        ],
        "@typescript-eslint/consistent-type-imports": [
            "error",
            {
                "fixStyle": "inline-type-imports"
            }
        ],
```

Looks like there's another setting we could use: https://typescript-eslint.io/rules/consistent-type-imports/#fixstyle

> I noticed that index.ts imports are using whole block which is the most important IMO. But I like whole block as default so I can see that there really isn't a runtime dep.

You still have to be careful to check for a second non-type-only export from the same module, right?

Reply:
Yes, you would unless we ban double import, which I think is pretty common.
What manual tweaks were you making (broadly speaking)?

Reply:
Opened #19479 to update the lint config to prefer block-level type-only exports/imports